frontend/src/components/activity/ContentType.vue

```diff
@@ -14,7 +14,7 @@
       @click="addActivityContent">
       <v-icon :left="$vuetify.breakpoint.smAndUp" size="150%">mdi-plus</v-icon>
 
-      {{ $tc('addButton', 1, { contentType: $t('activityContent.' + contentTypeName + '.name') }) }}
+      {{ $tc('addButton', 1, { contentType: $t(activityTypeNameKey) }) }}
```

Comment:
Should be `contentTypeNameKey`

Reply:
Ich habe diesen PR in #480 integriert, dort ist das gelöst bzw. kein Problem mehr